### PR TITLE
Updated the document to use the package from NuGet

### PR DESCRIPTION
**Pull request by shmao**

### CODE REVIEW

File: samples/xmlserializergenerator-instructions.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ You can start using the tool today following the instructions below.
 ## Prerequisites
```
Comment (zhenlan) [inline samples/xmlserializergenerator-instructions.md]: Can we add a note something like below?

"The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed."

File: samples/xmlserializergenerator-instructions.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ You can start using the tool today following the instructions below.
...
 
 * [.Net Core SDK of version 2.0.2 or later](https://www.microsoft.com/net/download/windows)
```
Comment (zhenlan) [inline samples/xmlserializergenerator-instructions.md]: Maybe just ".Net Core SDK 2.0.2 or later"?

File: samples/xmlserializergenerator-instructions.md [hunk 3]
```diff
@@ -17,24 +17,15 @@ Here are the step by step instructions on how to use Xml Serializer Generator in
...
-    <?xml version="1.0" encoding="utf-8"?>
-    <configuration>
-      <packageSources>
-        <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-      </packageSources>
-    </configuration>
-    ```
-3. Edit the .csproj and add a reference to the Microsoft.XmlSerializer.Generator package. For example,
+2. Edit the .csproj and add a reference to the Microsoft.XmlSerializer.Generator package. For example,
     * Run command: `dotnet add package Microsoft.XmlSerializer.Generator -v 1.0.0-preview1-25915-02`
```
Comment (zhenlan) [inline samples/xmlserializergenerator-instructions.md]: Can this be changed to 2.1 and 2.2?

File: samples/xmlserializergenerator-instructions.md [hunk 4]
```diff
@@ -45,6 +36,6 @@ Here are the step by step instructions on how to use Xml Serializer Generator in
     ```c#
     var serializer = new System.Xml.Serialization.XmlSerializer(typeof(MyClass));
     ```    
-5. Build the application by running `dotnet build`. If everything succeeds, an assembly named MyApp.XmlSerializers.dll will be generated    in the output folder. You will see warnings in the build output if the tool failed to generate the assembly.
+4. Build the application by running `dotnet build`. If everything succeeds, an assembly named MyApp.XmlSerializers.dll will be generated    in the output folder. You will see warnings in the build output if the tool failed to generate the assembly.
```
Comment (huanwu) [inline samples/xmlserializergenerator-instructions.md]: Remove the space between generated  "will be generated    in the output folder"

File: samples/xmlserializergenerator-instructions.md (hunk 1 above)
Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Added the above note.

File: samples/xmlserializergenerator-instructions.md (hunk 2 above)
Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Done

File: samples/xmlserializergenerator-instructions.md (hunk 3 above)
Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Done. Change this to 
> i.

File: samples/xmlserializergenerator-instructions.md (hunk 4 above)
Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Done.

File: samples/xmlserializergenerator-instructions.md [hunk 5]
```diff
@@ -6,8 +6,10 @@ You can start using the tool today following the instructions below.
 
 ## Prerequisites
 
-* [.Net Core SDK of version 2.0.2 or later](https://www.microsoft.com/net/download/windows)
-* [.NET Core runtime 2.0.3](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
+The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+* [.Net Core SDK 2.0.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.0.3 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
```
Comment (zhenlan) [inline samples/xmlserializergenerator-instructions.md]: Just noticed, we had different casing of ".NET" above.

Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Good catch. I'll fix it.

Comment (shmao) [reply samples/xmlserializergenerator-instructions.md]: Changed all ".Net" to ".NET" throughout the doc.

**Pull request merged**